### DISCUSSION TITLE
使用 客户端或是 Javamail 进行配置发送邮件不成功？

### DISCUSSION THREAD

**Discussion by jackyglony** [Jinnrry/PMail#180] [Q&A]
大神， 您好！很感谢您的开源邮件服务器。
  在使用过程中有两个疑问
  1. 部署时设置用户是否为邮箱用户？这里再使用JavaMail 发送邮件时需要用户和密码 
  2. 再对应 pop ,smtp 协议ping ，telnet 都能通过的情况下，配置第三方邮件客户端（网易邮箱大师）一直不成功，不知道是否还有其配置需要修改？
   
再次感谢 ！！

**Comment by Jinnrry** [comment]
1、是邮箱用户，而且是管理员账号（可以收到未匹配收信账号的全部邮件）
2、不成功有什么提示吗？有错误日志啥的吗？只说一个不成功我也没法帮你看是啥原因呀。
